### ISSUE TITLE
Discussion: I don't think we should accept social media related apps.

### ISSUE THREAD

**Issue by alichtman** [jaywcjlove/awesome-mac#357]
This is a repository targeted at designers and developers, not social media consumers. While there may be cool software out there for social media sites, I don't think it belongs in this list.

Opinions on removing those?

**Comment by xcv58** [comment]
Some social media apps did give me the meaningful idea/thought about how decent app is built.

**Comment by jaywcjlove** [comment]
Now we have become very big, Different from the original idea.

> Collect premium software in various categories.

**Comment by 8times9** [comment]
@jaywcjlove That's a great point and I'd definitely agree this list has become filled with apps that have little to do with development or content creation. 

For example, there's an entire section for gaming (albeit with only one app, OpenEmu) and apps like Popcorn Time have absolutely nothing to do with what this repo was created for.

However, I'd argue the main issue is the many apps that may serve useful for developers, but could also be equally useful for anyone else using them. Apps like Rocket or even some of my contributions, Shifty and SensibleSideButtons, may be useful for developers, but they are not explicitly related to content creation/development.

**Comment by alichtman** [comment]
> I'd argue the main issue is the many apps that may serve useful for developers, but could also be equally useful for anyone else using them.

@8times9 I completely agree. What are your thoughts on limiting this repo to solely apps that could aid in the development of content? This would mean that Social Media and Gaming apps, for example, would be removed.

**Comment by xcv58** [comment]
I don't think there is a clear line between apps for **developers and designers** and others. In my opinion, the Note-taking, Ebooks, RSS, Download Management Tools, To-Do Lists, Password Management is not related to **developers and designers**.

What's the difference between the Kindle App and a 3rd party social media client of the purpose for **developers and designers**? I would argue one can use the social media to get the latest tech news if you argue one can use Kindle to read tech book.

I do agree with you that the findability is crucial as the list become longer. Can we think and work on a better solution without limiting the category?

**Comment by alichtman** [comment]
Check out #114 for some previous brainstorming we did.

**Comment by 8times9** [comment]
The only objection I would have to removing unrelated apps would be that many genuinely useful apps might be lost, or at least made more difficult to find. I have a few ideas:

1. Clone this repo/make a new repo and have the it be focused around useful apps for anyone. Pull requests that don't fit here could be redirected there (best IMO).

2. Clone this repo/make a new repo and have it be focused on development and designing. 

3. Find another repo that gathers useful Mac apps for anyone and ensure all unrelated apps here are added there before being removed.

**Comment by xcv58** [comment]
@8times9 it seems very hard to maintain.

**Comment by 8times9** [comment]
[This](https://github.com/iCHAIT/awesome-macOS) seems to be a great repo to transfer apps to should we carry out the third plan.

**Comment by jaywcjlove** [comment]
I propose to change our original intention. 

Because it has become very big.

This is because we are constantly updating and we stay active and rich content. Won many stars.

**Comment by alichtman** [comment]
> Find another repo that gathers useful Mac apps for anyone and ensure all unrelated apps here are added there before being removed.

> This seems to be a great repo to transfer apps to should we carry out the third plan.

100% on board with this.
